app/models/knight.rb

```diff
@@ -1 +1,55 @@
-class Knight < Piece; end
+class Knight < Piece
+  def valid_move?(destination_x, destination_y)
+    return true if up_one_right_two?(destination_x, destination_y)
+
+    return true if up_two_right_one?(destination_x, destination_y)
+
+    return true if up_one_left_two?(destination_x, destination_y)
+
+    return true if up_two_left_one?(destination_x, destination_y)
+
+    return true if down_one_left_two?(destination_x, destination_y)
+
+    return true if down_two_left_one?(destination_x, destination_y)
+
+    return true if down_two_right_one?(destination_x, destination_y)
+
+    return true if down_one_right_two?(destination_x, destination_y)
+
+    false
+  end
```

Comment:
This format looks good. Is very readable (reads just like english) 👍 .

Re: case statements. I'd have to see it to decide. But I think what you did here works well.

Generally I think either way is fine. Rubocop is just a tool to help keep our code consistent/clean. If we find it works against us, we can always configure it to work better (as we have already quite a bit in the rubo config)